File: AutoReser - Angular/src/app/IzvjestajiVM.ts
export class Izvjestaj {
  izvjestajID!: number;
  naziv!: string;
  opis!: string;
  automobilID!: string;
  uposlenikKreiraIzvjestaj!: string;
  datumIzvjestaja!: string;
  korisnickiNalogID!: string;
}

export class IzvjestajiVM{
  izvjestaji!: Izvjestaj[];
  q?: any;
  total!: number;
}

File: RS1_seminarski/Modelview/IzvjestajiVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace RS1_seminarski.Modelview
{
    public class IzvjestajiVM
    {
        public int total { get; set; }
        public List<Izvjestaj> izvjestaji { get; set; }
        public string q { get; set; }
        public class Izvjestaj
        {
            public int izvjestajID { get; set; }
            public int ukupnoZapisa { get; set; }
            public string naziv { get; set; }
            public string opis { get; set; }
            public string automobilID { get; set; }
            public string uposlenikKreiraIzvjestaj { get; set; }

            public string datumIzvjestaja { get; set; }

            public string korisnickiNalogID { get; set; }
        }
    }
}
